pkgs/development/python-modules/aiolifx_effects/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv,
+  fetchPypi,
+  buildPythonPackage,
+  aiolifx
+}:
+
+buildPythonPackage rec {
+  pname = "aiolifx_effects";
```

Comment:
```suggestion
  pname = "aiolifx-effects";
```

Reply:
I originally did that, but I would get a 404 error when using a `-`. Did you have success with this syntax ?

Reply:
You have to add `pname = "aiolifx_effects";` inside `fetchPypi`.